README.md

```diff
@@ -317,6 +317,20 @@ plugins:
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins-ci.org/secure/Dashboard.jspa?selectPageId=17346) for known compatibility issues.
 
+## Triggering Configuration Reload
+
+You have the following option to trigger a configuration reload:
+
+- via the user interface: `Manage Jenkins -> Configuration -> Reload existing configuration`
+- via http POST to `JENKINS_URL/configuration-as-code/reload`
+  Note: this needs to include a valid CRUMB and authentication information e.g. username + token of a user with admin
+  permissions
+- via Jenkins CLI
+- via htto POST to `JENKINS_URL/reload-configuration-as-code/`
+  In contrast to the above it does not need authentication and bypasses CSRF checks, but it needs to be explicitly
```

Comment:
I think an API call is probably reasonable that doesn't require csrf.
But if security is enabled I would think security should be enforced.
You should be able to reload without authentication if you disable auth

Reply:
If it's not explicitly activated CSRF is enforced. see https://github.com/jenkinsci/configuration-as-code-plugin/pull/712/files#diff-eff409bf85cde6285ce608c11ccbb09bR18